src/components/OldApp/OldApp.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import WebcamCaptureContainer from '../WebcamCapture/WebcamCaptureContainer.js';
+import ConfirmationBox from '../ConfirmationBox/ConfirmationBox';
+import StoreListContainer from '../StoreList/StoreListContainer';
+import PropTypes from 'prop-types';
+import {getUserSlackID, sendSlackMessage} from './../../utils/slack';
+import UsernameEntryContainer from '../UsernameEntry/UsernameEntryContainer.js';
+import Notification from './../Notification/Notification';
+import './OldApp.css';
+
+const NOTIFICATION_DURATION = 5000;
+
+class App extends React.Component {
+	state = {
+		showNotification: false,
+		notificationMessage: '',
+		isError: false
+	};
+
+	getStoreCode = name => {
+		for (let item in this.props.storeList) {
```

Comment:
Noticing that we're using `let` in a bunch of places where we could use `const`, could we add a PR (can be separate piece of work) to add the linting rule so this is flagged up by default?